src/pages/task/TestsTableCore.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { useEffect } from "react";
+import { ColumnProps, TableProps } from "antd/es/table";
+import { InfinityTable } from "antd-table-infinity";
+import { msToDuration } from "utils/string";
+import { Tag, Spin } from "antd";
+import { TESTS_QUERY } from "utils/gql/queries";
+import { useParams, useLocation, useHistory } from "react-router-dom";
+import { useQuery } from "@apollo/react-hooks";
+import { usePrevious } from "utils/hooks";
+import {
+  Categories,
+  RequiredQueryParams,
+  Sort,
+  ValidInitialQueryParams,
+  Limit
+} from "pages/task/types";
+import get from "lodash.get";
+import queryString from "query-string";
+
+interface TaskTests {
+  id: string;
+  status: string;
+  testFile: string;
+  duration: number;
+}
+
+interface UpdateQueryArg {
+  taskTests: [TaskTests];
+}
+
+const loadMoreContent = (): JSX.Element => (
+  <div
+    style={{
```

Comment:
no inline styles please. should use emotion for all styling to be consistent

Reply:
This looks like a React element but is named as a function. The name also threw me off--when i first read it i thought it was going to be a function to load more content. But it's a loading component